service/notification/NotificationProcessor.php

```diff
@@ -206,7 +203,7 @@ public function processNotification($unprocessedNotification)
                 
                 break;
             case AdyenNotification::OFFER_CLOSED:
-                // Notification success is 'true'
+                // Notification success is 'true' AND current status is a non-final one
```

Comment:
```suggestion
                // Notification success is 'true' AND current status is ADYEN_OS_WAITING_FOR_PAYMENT
```

Offer_closed should not cancel even other statuses only the waiting for payment one